lib/app/worker.rb

```diff
@@ -19,9 +19,26 @@ module Worker
 
     class << self
       def start!
+        disable_warnings = if App::Config.has_key?(:disable_warnings)
+                             App::Config[:disable_warnings]
```

Comment:
Should add to [connectors-ruby/config/connectors.yml.example](https://github.com/elastic/connectors-ruby/blob/main/config/connectors.yml.example)